gnovm/pkg/gnomod/gnomod.go

```diff
@@ -96,11 +87,12 @@ func writePackage(remote, basePath, pkgPath string) (requirements []string, err
 // GnoToGoMod make necessary modifications in the gno.mod
 // and return go.mod file.
 func GnoToGoMod(f File) (*File, error) {
+	// TODO(morgan): good candidate to move to pkg/transpiler.
```

Comment:
Leftover?

Reply:
Not, it is quite out of scope here but it would be nice. It's a refactor that I think should be tackled by me or the next person annoyed enough at this code to refactor it; but not as part of this PR.